pyproject.toml

```diff
@@ -54,13 +54,14 @@ exclude = []
 
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
-SQLAlchemy = ">=1.3.24,<1.4"
+SQLAlchemy = ">=1.4"
 pyodbc = ">=4.0.34"
 packaging = "^21.3"
 # TODO: Reenable once the causing issues got fixed:
 #  - https://github.com/blue-yonder/turbodbc/issues/358
 #  - https://github.com/exasol/sqlalchemy-exasol/issues/146
 # turbodbc = { version = ">4.5.3", optional = true }
+turbodbc = {path = "turbodbc-4.5.3.tar.gz"}
```

Comment:
This needs to go later

Reply:
Yes, good reminder. But need it for testing right now (at least until https://github.com/blue-yonder/turbodbc/pull/359 gets merged and released).